Microsoft.PowerShell_profile.ps1

```diff
@@ -9,13 +9,30 @@ function prompt {
 $Host.PrivateData.DebugBackgroundColor = "DarkGray"
 $Host.PrivateData.ErrorBackgroundColor = "DarkGray"
 
-Set-PSReadlineOption -TokenKind Command   -ForegroundColor White
-Set-PSReadlineOption -TokenKind Comment   -ForegroundColor DarkGreen
-Set-PSReadlineOption -TokenKind Keyword   -ForegroundColor Blue
-Set-PSReadlineOption -TokenKind Member    -ForegroundColor Cyan
-Set-PSReadlineOption -TokenKind Number    -ForegroundColor Yellow
-Set-PSReadlineOption -TokenKind Operator  -ForegroundColor Magenta
-Set-PSReadlineOption -TokenKind Parameter -ForegroundColor Cyan
-Set-PSReadlineOption -TokenKind String    -ForegroundColor Yellow
-Set-PSReadlineOption -TokenKind Type      -ForegroundColor Green
-Set-PSReadlineOption -TokenKind Variable  -ForegroundColor White
+
+$promptColors = @{
```

Comment:
I'm not an expert of posh, but we should be able to use this hashtable for both if-else branches.